mitodl/mit-open
# Add load testing suite

### What are the relevant tickets?




Related to https://github.com/mitodl/hq/issues/5101

### Description (What does it do?)

This adds Locust as a load testing tool. It also removes some configuration around uwsgi processes and threads because this doesn't seem to be getting picked up properly.

### How can this be tested?
- Add `load-testing` to your `COMPOSE_PROFILES` setting and `docker compose up`
- Navigate to http://localhost:8089/
- Enter some load test settings (e.g. 100 users), verify it's running against the nginx hostname, and run it. You should see Locust running load tests.
